src/main/kotlin/com/rustyrazorblade/easycassstress/RequestQueue.kt

```diff
@@ -113,8 +95,6 @@ class RequestQueue(
                             runner.getNextMutation(key)
                         }
 
-                    op.startTime = getTimer(op).time()
```

Comment:
so start is based off queue time, i changed it to be based off request time... i prob should make this 2 timestamps